data_steward/tools/consolidated_achilles_report/get_all_results_html.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import gcs_utils
+import json
+import os
+import query_reports
+
+DRC_BUCKET_PATH = 'gs://%s/' % gcs_utils.get_drc_bucket()
+
+
+def get_hpo_id(p):
+    rel_path = p[len(DRC_BUCKET_PATH):]
+    return rel_path[:rel_path.index('/')]
+
+
+def transform_bq_list(uploads):
+    """
+    Get paths to all most recent report files
+
+    :return:
+    """
+    results = []
+    for upload in uploads:
+        dte, p = upload['upload_timestamp'], upload['file_path']
+        hpo_id = get_hpo_id(p)
+        report_path = p.replace('person.csv', 'results.html')
+        result = dict(hpo_id=hpo_id, updated=dte, report_path=report_path)
+        results.append(result)
+    return results
+
+
+def download_report(s):
+    """
+    Download most recent report files
+    :param s:
```

Comment:
Can we use a more descriptive variable name than 's'?  And please add the documentation definition for 's' here.